lib/providers/event_providers.dart

```diff
@@ -0,0 +1,41 @@
+import 'dart:async';
+
+import 'package:firebase_database/firebase_database.dart';
+import 'package:regatta_buddy/providers/firebase_providers.dart';
+import 'package:regatta_buddy/utils/logging/logger_helper.dart';
+import 'package:riverpod_annotation/riverpod_annotation.dart';
+
+part 'event_providers.g.dart';
+
+@riverpod
+Stream<Map<String, List<int>>> teamScore(TeamScoreRef ref, String eventId) {
```

Comment:
Troche nie łapie, to jest dla jednego teamu czy dla wszystkich teamow w evencie?

Reply:
wszystkie teamy (key) w wydarzeniu i punkty zdobyte w kolejnych rundach (value)

Reply:
jak na moje zmień na team scores wtedy, teamScoreProvider brzmi jakby miał przyjmować teamId